client/packages/cli/config/setup.ts

```diff
@@ -40,11 +44,39 @@ export default {
                     relayChainId: "roco",
                     id: 1000
                 },
-                gatewayVendor: "Rococo",
-                gatewaySysProps: {
-                    tokenSymbol: "ROC",
-                    tokenDecimals: 12,
-                    ss58Format: 42
+                verificationVendor: "Rococo",
+                executionVendor: "Substrate",
+                runtimeCodec: "Scale",
+                tokenInfo: {
+                    Substrate: {
+                        symbol: "ROC",
+                        decimals: 12,
+                        id: 42
+                    }
+                },
+                allowedSideEffects: [["tran", 4]],
+            }
+        },
+        {
+            name: "Ethereum",
+            id: "eth2",
+            token_id: "eth2",
```

Comment:
```suggestion
            tokenId: "eth2",            
```

Assuming camelCase here too.